shim/include/kvm_regs.h

```diff
@@ -44,8 +44,42 @@ extern "C"
      */
     struct kvm_regs
     {
-        /** @brief replace me with contents from KVM API */
-        int32_t dummy;
+        /** @brief rax stores register value */
+        uint64_t rax;
```

Comment:
change these to:
"stores that value of the rax register"

This way we can use the same language for MSRs, etc.